components/plans/create/AttributePicker.tsx

```diff
@@ -0,0 +1,231 @@
+import { PlusIcon, XMarkIcon } from "@heroicons/react/20/solid";
+import clsx from "clsx";
+import { useState } from "react";
+import { useMap } from "react-map-gl";
+import Button from "../../Button";
+import CardSlider from "../../CardSlider";
+
+interface AttributesPickerProps {
+  attributes: string[] | undefined;
+  selectedAttributes: string[][];
+  setSelectedAttributes: (stops: string[][]) => void;
+  attributesIndex: number;
+  setAttributesIndex: (index: number) => void;
+  setSelectedTabIndex: (index: number) => void;
+  suggestions: Venue[] | undefined;
+}
+
+export default function AttributesPicker({
+  attributes,
+  selectedAttributes,
+  setSelectedAttributes,
+  attributesIndex,
+  setAttributesIndex,
+  setSelectedTabIndex,
+  suggestions,
+}: AttributesPickerProps) {
+  const { map } = useMap();
```

Comment:
I would look to use React Query here to get 'map' instead of the current hook implementation. This is something we could go over in a call if you like. It has benefits like caching, query invalidation. You can pull out loads of useful stuff form the hooks they provide.